scripts/setup-breaches.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-env node */
+
+const fs = require("fs");
+const path = require("path");
+const fetch = require("node-fetch");
+
+async function download(url, relativeDestPath) {
+  const contents = await (await fetch(url)).text();
+  return new Promise((resolve, reject) => {
+    fs.writeFile(path.join(path.dirname(__dirname), relativeDestPath), contents, (err) => {
+      if (err) {
+        reject(err);
+        return;
+      }
+      resolve(contents);
+    });
+  });
+}
+
+async function run() {
+  const breachFileText = await download(
+    "https://haveibeenpwned.com/api/v2/breaches",
+    path.join("src", "breaches.json"));
```

Comment:
This is potentially interesting.
Instead of pulling a list of recent breaches from the HIBP server, should we be pulling from the blurts-server? I mean, there is a _slight_ possibility that we eventually have to release a v1.1 or v1.0.1 (or whatever) and then then the add-on pulls in a recent breach that the server isn't aware of, and the link would 404 and just oddly redirect to the homepage. By making the blurts-server the "source of truth(tm)", we may keep things slightly more in sync.